server_installation/topics/operator/keycloak-backup-cr.adoc

```diff
@@ -0,0 +1,106 @@
+
+=== KeycloakBackup Custom Resource
+
+{project_operator} provides automatic backups with manual restore in 3 modes:
+
+* One time backups to a local Persistent Volume.
+* One time backups to Amazon S3 storage.
+* Periodic backups to Amazon S3 storage.
+
+The Operator uses `KeycloakBackup` Custom Resource to trigger a backup Job (or a `CronJob` in case of Periodic Backups) and reports back its status. The CR has the following structure:
+
+.`KeycloakBackup` Custom Resource
+```yaml
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakBackup
+metadata:
+  name: <CR Name>
+spec:
+  aws:
+    # Optional - used only for Periodic Backups
+    schedule: <Cron Job Schedule>
+    # Requred - used to obtain credentials for storing backup in S3
+    credentialsSecretName: <A Secret containing S3 credentials>
+```
+
+.AWS S3 `Secret`
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: <Secret Name>
+type: Opaque
+stringData:
+  AWS_S3_BUCKET_NAME: <S3 Bucket Name>
+  AWS_ACCESS_KEY_ID: <AWS Access Key ID>
+  AWS_SECRET_ACCESS_KEY: <AWS Secret Key>
+```
+
+The above secret name needs to match the one referred in the `KeycloakBackup` Custom Resource.
+
+Once the `KeycloakBackup` Custom Resource is created, {project_operator} will create a corresponding `Job`
+responsible for backing up Postgresql database. The backup status will be reported in the status field.
```

Comment:
Maybe "will create a corresponding `Job` to back up the PostgreSQL database." ?
Maybe "The status of the backup is reported in the `Status` field."

Reply:
Fixed.